src/palindrom.js

```diff
@@ -780,6 +781,16 @@ var Palindrom = (function() {
         );
       }
     });
+    /* when patches are applied, and their `value`s are objects, make sure to pass the proxified version */
```

Comment:
Replied in https://github.com/Palindrom/Palindrom/pull/135#issuecomment-300544893 (not to be hidden by GH)